document_page_environmental_aspect/__manifest__.py

```diff
@@ -32,6 +32,7 @@
     "data": [
         'data/document_page.xml',
         'views/document_page.xml',
+        'security/environmental_aspect_security.xml',
     ],
     'installable': False,
```

Comment:
@ernestotejeda This module is `installable` False so not sure if I am missing something but any changes aren't notice for Travis for instance